{"version":3,"sources":["components/ConferenceCallAlert/i18n/pt-BR.js"],"names":["conferenceCallErrors","bringInFailed","makeConferenceFailed","terminateConferenceFailed","removeFromConferenceFailed"],"mappings":";;;;;;;;;;;;AAAA;;;;;;oGAGGA,+BAAqBC,a,EAAgB,sF,wDACrCD,+BAAqBE,oB,EAAuB,sF,wDAC5CF,+BAAqBG,yB,EAA4B,yF,wDACjDH,+BAAqBI,0B,EAA6B,yF;;AAGrD;AACA;AACA;AACA","file":"pt-BR.js","sourcesContent":["import conferenceCallErrors from 'ringcentral-integration/modules/ConferenceCall/conferenceCallErrors';\n\nexport default {\n  [conferenceCallErrors.bringInFailed]: \"Falha ao mesclar as chamadas devido a erros inesperados. Tente novamente mais tarde.\",\n  [conferenceCallErrors.makeConferenceFailed]: \"Falha ao mesclar as chamadas devido a erros inesperados. Tente novamente mais tarde.\",\n  [conferenceCallErrors.terminateConferenceFailed]: \"Falha ao desligar a conferÃªncia devido a erros inesperados. Tente novamente mais tarde.\",\n  [conferenceCallErrors.removeFromConferenceFailed]: \"Falha ao remover o participante devido a erros inesperados. Tente novamente mais tarde.\"\n};\n\n// @key: @#@\"[conferenceCallErrors.bringInFailed]\"@#@ @source: @#@\"Failed to merge the calls due to unexpected errors. Please try again later.\"@#@\n// @key: @#@\"[conferenceCallErrors.makeConferenceFailed]\"@#@ @source: @#@\"Failed to merge the calls due to unexpected errors. Please try again later.\"@#@\n// @key: @#@\"[conferenceCallErrors.terminateConferenceFailed]\"@#@ @source: @#@\"Failed to hangup the conference due to unexpected errors. Please try again later.\"@#@\n// @key: @#@\"[conferenceCallErrors.removeFromConferenceFailed]\"@#@ @source: @#@\"Failed to remove the participant due to unexpected errors. Please try again later.\"@#@\n"]}